app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt

```diff
@@ -173,7 +201,14 @@ fun BluromaticScreenContentPreview() {
             blurUiState = BlurUiState.Default,
```

Comment:
Move this method below the private method

`private fun showBlurredImage(context: Context, currentUri: String) {`

Reply:
Moved.